### PR TITLE
Ran a full, non-incremental, yapf on the whole project.

### PR DESCRIPTION
**Pull request by bwbeach**

### CODE REVIEW

File: b2/raw_simulator.py [hunk 1]
```diff
@@ -554,12 +554,12 @@ def get_download_authorization(
...
-            bucket_id,
+                bucket_id,
             'fileNamePrefix':
-            file_name_prefix,
+                file_name_prefix,
             'authorizationToken':
-            'fake_download_auth_token_%s_%s_%d' %
-            (bucket_id, b2_url_encode(file_name_prefix), valid_duration_in_seconds)
+                'fake_download_auth_token_%s_%s_%d' %
+                (bucket_id, b2_url_encode(file_name_prefix), valid_duration_in_seconds)
```
Comment (ppolewicz) [inline b2/raw_simulator.py]: this gets poorly formatted due to missing , at the end of last value of multiline dict, please correct it

File: b2/sync/policy.py [hunk 2]
```diff
@@ -170,8 +169,8 @@ class UpPolicy(AbstractFileSyncPolicy):
     def _make_transfer_action(self):
         return B2UploadAction(
             self._source_folder.make_full_path(self._source_file.name), self._source_file.name,
-            self._dest_folder.make_full_path(self._source_file.name),
-            self._get_source_mod_time(), self._source_file.latest_version().size
+            self._dest_folder.make_full_path(self._source_file.name), self._get_source_mod_time(),
+            self._source_file.latest_version().size
```
Comment (ppolewicz) [inline b2/sync/policy.py]: this gets poorly formatted due to missing , at the end of last value, please correct it

File: b2/sync/policy.py [hunk 3]
```diff
@@ -152,11 +152,10 @@ class DownPolicy(AbstractFileSyncPolicy):
...
     def _make_transfer_action(self):
         return B2DownloadAction(
-            self._source_file.name,
-            self._source_folder.make_full_path(self._source_file.name),
+            self._source_file.name, self._source_folder.make_full_path(self._source_file.name),
             self._source_file.latest_version().id_,
-            self._dest_folder.make_full_path(self._source_file.name),
-            self._get_source_mod_time(), self._source_file.latest_version().size
+            self._dest_folder.make_full_path(self._source_file.name), self._get_source_mod_time(),
+            self._source_file.latest_version().size
```
Comment (ppolewicz) [inline b2/sync/policy.py]: this gets poorly formatted due to missing , at the end of last value, please correct it

File: b2/sync/policy.py [hunk 4]
```diff
@@ -246,8 +245,7 @@ def make_b2_delete_actions(source_file, dest_file, dest_folder, transferred):
         keep = (version_index == 0) and (source_file is not None) and not transferred
         if not keep:
             yield B2DeleteAction(
-                dest_file.name,
-                dest_folder.make_full_path(dest_file.name), version.id_,
+                dest_file.name, dest_folder.make_full_path(dest_file.name), version.id_,
                 make_b2_delete_note(version, version_index, transferred)
```
Comment (ppolewicz) [inline b2/sync/policy.py]: this gets poorly formatted due to missing , at the end of last value, please correct it

File: b2/sync/policy.py [hunk 5]
```diff
@@ -299,8 +297,7 @@ def make_b2_keep_days_actions(
         # Delete this version
         if deleting:
             yield B2DeleteAction(
-                dest_file.name,
-                dest_folder.make_full_path(dest_file.name), version.id_,
+                dest_file.name, dest_folder.make_full_path(dest_file.name), version.id_,
                 make_b2_delete_note(version, version_index, transferred)
```
Comment (ppolewicz) [inline b2/sync/policy.py]: this gets poorly formatted due to missing , at the end of last value, please correct it

File: doc/source/conf.py [hunk 6]
```diff
@@ -112,13 +112,14 @@
...
-        'donate.html',
-    ]
+    '**':
+        [
+            'about.html',
+            'navigation.html',
+            'relations.html',  # needs 'show_related': True theme option to display
+            'searchbox.html',
+            'donate.html',
+        ]
```
Comment (ppolewicz) [inline doc/source/conf.py]: this gets poorly formatted due to missing , at the end of last value of multiline dict, please correct it

File: b2/sync/policy.py (hunk 2 above)
Comment (bwbeach) [reply b2/sync/policy.py]: So we should always put a comma after the last argument to a function?  That seems awkward.

Comment (bwbeach) [reply b2/sync/policy.py]: The formatting is nicer.  My preference is actually to have one argument per line on multi-line function calls.

**Pull request merged**